Katello/katello
# Fixes #34721 - Use dropdown for Traces & Errata REX actions

#### What are the changes introduced in this pull request?

* Changed the button to a dropdown with the actions that are in the kebab to make it like we have it on the packages page

#### Considerations taken when implementing this change?

* Made sure all tests run and the REX actions function like before. 

#### What are the testing steps for this pull request?

* Click the drop-down on the restart app button and make sure the actions work
* Verify there are not any console errors
